src/components/Authentication/Login.js

```diff
@@ -1,57 +1,137 @@
+
 import { Auth } from "aws-amplify";
-import Button from '@material-ui/core/Button';
 import { setUser } from '../../actions/UserActions'
 import { connect } from "react-redux";
 import { Helmet } from 'react-helmet';
 
 const styles = {
-    left: {
-        float: 'left'
-    },
-    socialIcon: {
-        marginTop: '5px',
-        marginRight: '8px',
-        width: '19px'
-    }
+  left: {
+    float: "left"
+  },
+  socialIcon: {
+    marginTop: "5px",
+    marginRight: "8px",
+    width: "19px"
+  }
 };
 
-function Login(props) {
-
-    Auth.currentAuthenticatedUser()
-        .then(user => {
-            const email = user.attributes.email
-            if (email.substring(email.indexOf("@") + 1, email.length) === 'ubcbiztech.com') {
-                props.setUser(user)
-            }
-            else {
-                Auth.signOut()
-                alert('You must use a ubcbiztech.com email')
-            }
-        })
-        .catch((err) => console.log("Not signed in", err))
+const useStyles = makeStyles(theme => ({
+  paper: {
+    marginTop: theme.spacing(10),
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+    // backgroundImage: `url(${BackgroundImage})`
+  },
+  avatar: {
+    margin: theme.spacing(1),
+    backgroundColor: "#54D260"
+  },
+  form: {
+    width: "100%", // Fix IE 11 issue.
+    marginTop: theme.spacing(1)
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2)
+  },
+  root: {
+    borderRadius: 12,
+    minWidth: 256,
+    textAlign: "center"
+  },
+  header: {
+    textAlign: "center",
+    spacing: 10
+  },
+  list: {
+    padding: "20px"
+  },
+  button: {
+    margin: theme.spacing(1)
+  },
+  action: {
+    display: "flex",
+    justifyContent: "space-around"
+  }
+}));
 
-    return (
-        <div>
-            <Helmet>
-                <title>UBC BizTech - Log In or Sign Up</title>
-            </Helmet>
-            <h1> Login </h1>
-            <Button onClick={() => Auth.federatedSignIn({ provider: 'Google' }).then(() => Auth.currentAuthenticatedUser({ bypassCache: true }))} variant="contained" color="primary">
-                <div style={styles.left}>
-                    <img style={styles.socialIcon} alt="Google" src="./google.png" />
-                </div>
-                Sign In with Google
+function Login(props) {
+  const classes = useStyles();
+  Auth.currentAuthenticatedUser()
+    .then(user => {
+      const email = user.attributes.email;
+      if (
+        email.substring(email.indexOf("@") + 1, email.length) ===
+        "ubcbiztech.com"
+      ) {
+        props.setUser(user);
+      } else {
+        Auth.signOut();
+        alert("You must use a ubcbiztech.com email");
+      }
+    })
+    .catch(err => console.log("Not signed in:", err));
+  return (
+    <Container component="main" maxWidth="xs">
+      <CssBaseline />
+      <div className={classes.paper}>
+        <Card className={classes.root} width="400px">
+          <Helmet>
```

Comment:
I think it's best for visibility if we put the helmet as close to the top of the component as possible, so maybe next to/above the `CssBaseline`?